src/metabase/util/log/capture.cljc

```diff
@@ -140,10 +143,6 @@
      `(do ~@body)
      bindings)))
 
-(s/fdef with-log-messages-for-level
-  :args ::with-log-messages-for-level-args
-  :ret  any?)
```

Comment:
I still think `fdef` on a macro is still useful because the validation on macroexpansion is actually built in to Clojure. Since macroexpansion only happens on the JVM anyway can't we just make `::with-log-messages-for-level-args` and friends `#?:clj` so we can keep this the way it is but still remove the Cljs need for spec?